CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Added
 - Ruby 2.4.1 testing
 
+## [1.0.1] - 2018-04-23
```

Comment:
Please leave versioning to the maintainers for several reasons:
- PRs are not a FIFO queue, merge order is not guaranteed
- maintainers version when the change is released not when it has been submitted
- maintainers may disagree with you on the version number based on its impact.In this case removing this filtering would constitute a breaking change per our commitment to semver and therefore would need to be a major version release. You can read about our versioning conventions [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md)
- sometimes we lump multiple submissions into a single release when there are multiple in a day to reduce the need to rebase

Please see our [changelog guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md) for future refrence